src/python/tests/core/bot/tasks/trials_test.py

```diff
@@ -44,45 +44,52 @@ def test_no_effect_on_no_match(self):
     """Ensure that no additional flags are added if a binary has no trials."""
     self.mock.random.return_value = 0.0
     environment.set_value('APP_NAME', 'app_0')
-    trials.setup_additional_args_for_app()
+    trial_selector = trials.Trials(environment)
+    trial_selector.setup_additional_args_for_app(environment)
     self.assertEqual(environment.get_value('APP_ARGS'), '-x')
     self.assertIsNone(environment.get_value('TRIAL_APP_ARGS'))
 
   def test_trial_selected_one_option(self):
     """Ensure that the expected flags are added if a trial is selected."""
     self.mock.random.return_value = 0.3
     environment.set_value('APP_NAME', 'app_1')
-    trials.setup_additional_args_for_app()
+    trial_selector = trials.Trials(environment)
+    trial_selector.setup_additional_args_for_app(environment)
     self.assertEqual(environment.get_value('APP_ARGS'), '-x --a1')
     self.assertEqual(environment.get_value('TRIAL_APP_ARGS'), '--a1')
 
   def test_trial_not_selected(self):
-    """Ensure that no additional flags a trial was not selected."""
+    """Ensure no additional flags if a trial was not selected."""
     self.mock.random.return_value = 0.5
     environment.set_value('APP_NAME', 'app_2')
+    trial_selector = trials.Trials(environment)
```

Comment:
Here it seems there was a line missing before that calls trials.setup_additional_args_for_app()

Reply:
Yes, thanks for fixing that.